aws/table_aws_cloudformation_stack_set.go

```diff
@@ -0,0 +1,282 @@
+				Description: "Describes whether StackSets performs non-conflicting operations concurrently and queues conflicting operations.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags associated with stack.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCloudFormationStackSet,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCloudFormationStackSet,
+				Transform:   transform.From(cfnStackSetTagsToTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("StackSetName"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCloudFormationStackSet,
+				Transform:   transform.FromField("StackSetARN").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listCloudFormationStackSets(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := CloudFormationClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_cloudformation_stack_set.listCloudFormationStackSets", "connection_error", err)
+		return nil, err
+	}
+
+	if svc == nil {
+		// Unsupported region check
+		return nil, nil
+	}
+
+	// Limiting the results
+	maxLimit := int32(100)
+	if d.QueryContext.Limit != nil {
+		limit := int32(*d.QueryContext.Limit)
+		if limit < maxLimit {
+				maxLimit = limit
+		}
+	}
+	status := d.EqualsQualString("status")
+
+	input := &cloudformation.ListStackSetsInput{
+		MaxResults: &maxLimit,
+	}
+
+	if status != "" {
+		input.Status = types.StackSetStatus(status)
+	}
+
+	paginator := cloudformation.NewListStackSetsPaginator(svc, input, func(o *cloudformation.ListStackSetsPaginatorOptions) {
+		o.StopOnDuplicateToken = true
+	})
+	for paginator.HasMorePages() {
+		output, err := paginator.NextPage(ctx)
+		if err != nil {
+			plugin.Logger(ctx).Error("aws_cloudformation_stack_set.listCloudFormationStackSets", "api_error", err)
+			return nil, err
+		}
+
+		for _, stackSet := range output.Summaries {
+			d.StreamListItem(ctx, stackSet)
+			// Context can be cancelled due to manual cancellation or the limit has been hit
+			if d.RowsRemaining(ctx) == 0 {
+				return nil, nil
+			}
+		}
+
+	}
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getCloudFormationStackSet(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+
+	var name string
+
+	if h.Item != nil {
+		data := h.Item.(types.StackSetSummary)
+		name = *data.StackSetName
+	} else {
+		name = d.EqualsQuals["stack_set_name"].GetStringValue()
```

Comment:
```suggestion
		name = d.EqualsQuals["stack_set_name"].GetStringValue()
```
@ParthaI can we use `EqualsQualString` instead?

Reply:
Updated